src/mlpack/bindings/cli/CMakeLists.txt

```
@@ -50,11 +50,21 @@ if (BUILD_CLI_EXECUTABLES)
   add_executable(mlpack_${name}
     ${name}_main.cpp
   )
-  target_link_libraries(mlpack_${name}
-    mlpack
-    ${ARMADILLO_LIBRARIES}
-    ${COMPILER_SUPPORT_LIBRARIES}
-  )
+  # Build mlpack CLI binding binaries statically.
+  if(NOT BUILD_SHARED_LIBS)
+    target_link_libraries(mlpack_${name} -static
+      mlpack
+      ${ARMADILLO_LIBRARIES}
+      ${COMPILER_SUPPORT_LIBRARIES}
+   )
```

```suggestion
    )
```
Ah, missed this one last time.